test/cypress/integration/settings/your-psp-stripe-tasklist.cy.test.js

```diff
@@ -110,4 +115,67 @@ describe('Your PSP Stripe page', () => {
     cy.visit(`/account/${gatewayAccountExternalId}/your-psp/${credentialExternalId}`)
     cy.get('span').contains('Government entity document').should('have.attr', 'href', `/account/${gatewayAccountExternalId}/your-psp/${credentialExternalId}/government-entity-document`)
   })
+
+  describe('Bank details task', () => {
+    it('should click bank details task and display bank details page correctly', () => {
+      setupYourPspStubs({
+        bankAccount: false,
+        director: false,
+        vatNumber: false,
+        companyNumber: false,
+        responsiblePerson: false,
+        organisationDetails: false,
+        governmentEntityDocument: false
+      })
+
+      cy.setEncryptedCookies(userExternalId)
+      cy.visit(`/account/${gatewayAccountExternalId}/your-psp/${credentialExternalId}`)
+      cy.get('span').contains('Bank Details').click()
+      cy.get('#bank-details-form').should('exist')
+        .within(() => {
+          cy.get('input#account-number').should('exist')
+          cy.get('input#sort-code').should('exist')
+          cy.get('button').should('exist')
+          cy.get('button').should('contain', 'Save and continue')
+        })
+    })
+
+    it('should redirect back to the task List when valid bank details submitted', () => {
+      setupYourPspStubs({
+        bankAccount: false,
+        director: false,
+        vatNumber: false,
+        companyNumber: false,
+        responsiblePerson: false,
+        organisationDetails: false,
+        governmentEntityDocument: false
+      })
+
+      cy.setEncryptedCookies(userExternalId)
+      cy.visit(`/account/${gatewayAccountExternalId}/your-psp/${credentialExternalId}`)
```

Comment:
We can continue from the previous test.
Which means, we don't have to visit this page again.
We can go straight to the Bank details.